src/cloud/project/privatelink-service.md

```diff
@@ -152,7 +152,32 @@ To test the connection to the VPC endpoint service:
    * Closing connection 0
    ```
    {:.no-copy}
+   
```

Comment:
So the following two steps to check whether connection is established, and to check the package flow   are appended to the procedure  "To test the connection to the VPC endpoint service".  

Last step of that procedure is ```Run curl -v telnet://<endpoint-service-dns-url>:<port>/```

followed by success and failure output.


Do I have to check whether connection is established, if the  telnet request succeeds, or only if it fails.

Same thing with the packages flow.

Is checking for "established connection" and checking package flow always required?  Or should there be a L3 (### Troubleshooting heading before these steps?

Reply:
Well, it's my bad. Actually, the CURL cmd is to check the connectivity and the NETSTAT is to check if the system has an established connection on whichever port needed. I'll change it to port instead of established.